frontend/src/api/billing.js

```diff
@@ -13,8 +13,12 @@ const getSubscriptionInfo = async (teamId) => {
 }
 
 // Create a new subscription for a team
-const createSubscription = async (teamId) => {
-    return client.post('/ee/billing/teams/' + teamId).then(res => {
+const createSubscription = async (teamId, teamTypeId) => {
```

Comment:
Given `teamTypeId` is optional, it should be set to undefined (even though redundant) to notate this:

```suggestion
const createSubscription = async (teamId, teamTypeId = undefined) => {
```